stepik-union/test/org/stepik/plugin/actions/DirectivesUtilsTest.java

```diff
@@ -31,12 +35,12 @@ public String readTestFile(String fileName) {
     @Before
     public void setUp() throws Exception {
         for (int i = 1; i < 7; i++) {
-            String fileName = JAVA + Integer.toString(i) + ".txt";
+            String fileName = JAVA + "/" + Integer.toString(i) + ".txt";
             testsMap.put(JAVA + i, readTestFile(fileName));
         }
 
         for (int i = 1; i < 7; i++) {
-            String fileName = PYTHON + Integer.toString(i) + ".txt";
+            String fileName = PYTHON + "/" + Integer.toString(i) + ".txt";
```

Comment:
Как раз в винде и сломался тест, ведь java там использует не File.separator
Вот что написано в javaDocs:
 Before delegation, an absolute resource name is constructed from the given resource name using this algorithm:

    If the name begins with a '/' ('\u002f'), then the absolute name of the resource is the portion of the name following the '/'.
    Otherwise, the absolute name is of the following form:

        modified_package_name/name 

    Where the modified_package_name is the package name of this object with '/' substituted for '.' ('\u002e').